Cakes-Co/index.html

```diff
@@ -7,10 +7,108 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+     <link rel="stylesheet" href="style.css">
+     <link href="https://fonts.googleapis.com/css2?family=Pacifico&family=Poppins:wght@400;700&display=swap" rel="stylesheet">
 </head>
-
 <body>
     <!-- Add your markup here -->
+
+
+
+
+   
+<header class="header">
+    <div class="gray-cover"></div>
+    <h1 class="header-text"> The best cakes in the town <br>delivered to your door</h1>
+</header>
+
+
+
+<nav class="navbar">
+    <ul class="navlist">
+        <li> <a href="#home">Home</a></li>
```

Comment:
Are those links like `#home` or `#cakes`, to other places on this page? If those are links to other pages, then we should not use [fragments](https://www.simplehtmlguide.com/linking.php#:~:text=It%20is%20often%20usefull%20to%20link%20to%20other%20places%20on%20the%20same%20webpage%2C%20such%20as%20other%20sections%20or%20chapters%20further%20down%20the%20page.) here.